pyproject.toml

```diff
@@ -30,6 +30,17 @@ classifiers = [
 dynamic = ["version"]
 
 [project.optional-dependencies]
+launch = [
+    "hydra-core>=1.2.0",
+    "termcolor>=2.1.1",
+    "wandb>=0.13.7",
+    "mlflow>=2.1.1",
+    "pydantic>=1.10.2",
+    "imageio>=2.28.1",
+    "moviepy>=1.0.3",
+    "tqdm>=4.60.0",
+]
+
 all = [
```

Comment:
Should ideally use self-referential dependencies here instead of re-listing everything:
https://github.com/pypa/hatch/blob/master/docs/config/dependency.md#self-referential

Reply:
Great idea! Implemented it